### PR TITLE
Removed kafka topic check

### PR DESCRIPTION
**Pull request by MihaiSorinMoisa**
Updated Dockerfile
Updated circle.yml
Reworked the healthcheck
Updated the producer and consumer libraries

### CODE REVIEW

File: vendor/vendor.json [hunk 1]
```diff
@@ -3,105 +3,105 @@
...
-			"checksumSHA1": "6R+ptya8zWzpWfv27a9PdmH6uVo=",
+			"checksumSHA1": "mirNgwWRdRte0ku/g4u4fNbzQP0=",
 			"path": "github.com/Financial-Times/message-queue-gonsumer/consumer",
-			"revision": "4e173298922ba4c10721b1b5b3e0d35be7f162e9",
-			"revisionTime": "2017-03-20T10:34:32Z",
-			"version": "0.3.0",
-			"versionExact": "0.3.0"
+			"revision": "0a481ff94df47f766a74867475247eaf1ed40107",
+			"revisionTime": "2017-05-26T12:08:30Z",
+			"version": "0.4.0-xp-remove-kafka-topic-check-rc.1",
```
Comment (tosan88) [inline vendor/vendor.json]: I assume this will be updated before the PR is merged

Comment (MihaiSorinMoisa) [inline vendor/vendor.json]: Yes, once I test this in xp I will create a new release for the library and update the vendor file.

**Comment by davidbalazs93** [comment]
wait with this PR until https://github.com/Financial-Times/upp-next-video-annotations-mapper/pull/5 reaches PROD

**Comment by tamas-molnar** [comment]
Use version `1.1.0`

**Pull request merged**